# Double: Publish rate for the outbound trajectories to the control plugins.
# Should be matched to timestep size in TrajectoryPlan inputs
# Units: Hz
trajectory_publish_rate: 10.0

# String: Name of default control plugin. Should match field in TrajectoryPlan message
# Due to lack of existing plugin discovery mechanism this is the only control plugin
# which will be available to TrajectoryExecutor
# Units: N/a
default_control_plugin: "pure_pursuit_wrapper_node"

# String: Full path to default control plugin's trajectory input topic
# Units: N/a
default_control_plugin_topic: "/guidance/plugins/pure_pursuit/plan_trajectory"
